fe/fe-core/src/main/java/com/starrocks/common/ConfigBase.java

```diff
@@ -198,14 +201,33 @@ private void setFields() throws Exception {
         }
     }
 
-    public static void setConfigField(Field f, String confVal) throws IllegalAccessException, Exception {
+    private static void validateConfValue(Field f, String[] arrayArgs, String confVal)
+            throws InvalidConfException {
+        switch (f.getName()) {
+            case "authentication_chain":
+                Set<String> argsSet = new HashSet<>(Arrays.asList(arrayArgs));
+                if (!f.getType().getSimpleName().equals("String[]")
+                        || argsSet.size() != arrayArgs.length
+                        || !argsSet.contains("native")) {
+                    throw new InvalidConfException("'authentication_chain' configuration invalid, current value: "
```

Comment:
show more details? like "native must be in the list, and the vals cannot be duplicate"

Reply:
Invalid config will cause fe start failed, so users need to find the reason ASAP.

Reply:
sure